server/sandbox_run.go

```diff
@@ -20,6 +20,7 @@ import (
 
 // RunPodSandbox creates and runs a pod-level sandbox.
 func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest) (*pb.RunPodSandboxResponse, error) {
+	s.Update()
```

Comment:
Yeah, the logging should be using Error priority, not Warning as it is now.  Changing it.